src/pynwb/misc.py

```diff
@@ -137,6 +137,11 @@ class Units(DynamicTable):
         'resolution'
     )
 
+    waveforms_desc = 'Individual waveforms for each spike. If the dataset is three-dimensional, the third dimension ' \
+                     'shows the response from different electrodes that all observe this unit simultaneously. In this' \
+                     ' case, the `electrodes` column of this Units table should be used to indicate which electrodes ' \
+                     'are associated with this unit, and the electrodes dimension here should be in the same order as' \
+                     ' the electrodes referenced in the `electrodes` column of this table.'
```

Comment:
```suggestion
    waveforms_desc = ('Individual waveforms for each spike. If the dataset is three-dimensional, the third dimension ' 
                      'shows the response from different electrodes that all observe this unit simultaneously. In this' 
                      ' case, the `electrodes` column of this Units table should be used to indicate which electrodes ' 
                      'are associated with this unit, and the electrodes dimension here should be in the same order as' 
                      ' the electrodes referenced in the `electrodes` column of this table.')
```

Reply:
Minor: style-wise, PEP8 says we should break long lines by wrapping them in parentheses instead of using backslashes: https://www.python.org/dev/peps/pep-0008/#:~:text=Long%20lines%20can%20be%20broken,a%20backslash%20for%20line%20continuation.&text=(See%20the%20previous%20discussion%20on,such%20multiline%20with%2Dstatements.)

See also discussion on: https://stackoverflow.com/questions/1874592/how-to-write-very-long-string-that-conforms-with-pep8-and-prevent-e501